src/desktop/OpenRAP/managers/TelemetryManager.ts

```diff
@@ -88,7 +88,12 @@ export class TelemetryManager {
 
   async batchJob() {
     try {
-      const loggedInUser = await this.userSDK.getUserSession();
+      let loggedInUser;
+      this.userSDK.getUserSession().then(data => {
```

Comment:
This won't work
it can be
` let loggedInUser = ;
loggedInUser = await this.userSDK.getUserSession().catch(err => logger.error("<error message>"));
`